lib/nfc/protocols/mifare_classic.c

```diff
@@ -391,6 +405,36 @@ void mf_classic_reader_add_sector(
     }
 }
 
+bool mf_classic_block_to_value(int32_t* value, uint8_t* addr, const uint8_t* block) {
+    uint32_t v = *(uint32_t*)&block[0];
+    uint32_t v_inv = *(uint32_t*)&block[4];
+    uint32_t v1 = *(uint32_t*)&block[8];
+
+    bool val_checks =
```

Comment:
it's missing a `&& block[12] == block[14]`, as is `00000000 FFFFFFFF 00000000 00 FF F0 0F` is considered a valid value block

Reply:
Not according to the specification:

![image](https://user-images.githubusercontent.com/22538735/213935459-f5f571f3-a55a-4db2-b673-e97cc510d67b.png)

Reply:
Ok, I misunderstood your comment